.github/workflows/execution-performance.yaml

```diff
@@ -15,8 +15,7 @@ jobs:
         contains(github.event.pull_request.labels.*.name, 'CICD:run-e2e-tests') ||
         contains(github.event.pull_request.labels.*.name, 'CICD:run-execution-performance-test') ||
         contains(github.event.pull_request.labels.*.name, 'CICD:run-execution-performance-full-test') ||
-        github.event.pull_request.auto_merge != null) ||
-        contains(github.event.pull_request.body, '#e2e'
```

Comment:
what will happen now if someone specifies e2e label?

will it run determinator, and then run the test if that says it's needed? Or will it not run it?

Reply:
> what will happen now if someone specifies e2e label?

AFAICT, this line: `contains(github.event.pull_request.body, '#e2e'` is checking for the presence of "e2e" in the commit body, which is something that isn't really used anymore 😄 Figured I'd clean it up.

Or do you mean, what happens if someone specifies the `CICD:run-e2e-tests` label? In that case, it'll only run the job if a relevant change is detected. `CICD:run-execution-performance-test` or `CICD:run-execution-performance-full-test` will force the job to run.